Gruntfile.js

```diff
@@ -2,7 +2,7 @@ module.exports = function(grunt) {
   grunt.loadNpmTasks("grunt-contrib-jshint");
   grunt.loadNpmTasks("grunt-jscs");
 
-  var srcFiles = ["app/**/*.js", "server.js"];
+  var srcFiles = [ "./server.js", "./app/script.js", "./lib/joke.js", "./lib/random.js", "./lib/randomLetters.js", "./lib/piglatinify.js", "./lib/quote.js"];
```

Comment:
good catch! Can also use "lib/**/*.js" to look at all the files in the lib folder.